src/log-manager/src/logrotate/logrotate.conf

```diff
@@ -1,12 +1,15 @@
-/usr/local/pai/logs/*/*/*/*/*.* {
+/usr/local/pai/logs/*/*/*/*/*.log
+/usr/local/pai/logs/*/*/*/*/*.stdout
+/usr/local/pai/logs/*/*/*/*/*.stderr
+/usr/local/pai/logs/*/*/*/*/*.all
+{
     nomail
-    olddir /usr/local/pai/logs-backup
     weekly
```

Comment:
what is "weekly" for?

Reply:
Weekly means log file will be rotated weekly. So if there is a log file which exist more than one week, it will be rotated

Reply:
Change it to monthly for long running job